online-shop/src/app/home-page/home-page.component.html

```diff
@@ -0,0 +1,3 @@
+<p>Welcome to the shop</p>
```

Comment:
your home page should be /products as per the requirements. you can then drop the whole home-page directory.